File: python/rna-transcription/rna_transcription.py
def to_rna(dna):
  
  dna_to_rna = {"G": "C", "C": "G", "T": "A", "A": "U"}
  
  try:
    return "".join(dna_to_rna[x] for x in dna)
  except KeyError:
    return ""

File: javascript/rna-transcription/rna-transcription.js
var DnaTranscriber = function() {};

rna_map = { "G":"`C", "C":"G", "T":"A", "A":"U" };

DnaTranscriber.prototype.toRna = function(dna) {

  var rna = "";

  for (var i = 0; i < dna.length; i++) {
    if (dna.charAt(i) in rna_map) {
      rna += rna_map[dna.charAt(i)];  
    }
    else {
      throw "Invalid input";
    }
  }

  return rna;

};

module.exports = DnaTranscriber;
